### ISSUE TITLE
v2.16 (Windows): Invalid algorithm mined

### ISSUE THREAD

**Issue by UselessGuru** [sp-hash/TeamBlackMiner#436]
`TeamBlackMiner-v2.16\TBMiner.exe --algo firopow --hostname firopow.eu.mine.zpool.ca --wallet ******************** --port 1326 --server-passwd ******************,c=BTC --api --api-version 1.4 --api-port 4005 --cl-devices [1]`

Instead of Firo it is mining Kawpow:

```
OpenCL driver version: 351.0
Cuda driver version: 12.3
Cuda runtime API version: 12.0
 ______ _____ _______
|_    _| __  |       |
  |  | | __ -| |  |  |
  |_ | |____ |_|__|  |

Team Black Miner v2.16


GPU0 Ellesmere 8GB added to miner as OpenCL device
Trying firopow.eu.mine.zpool.ca:1326 (kawpow)
Api server started at 127.0.0.1:4005
firopow.eu.mine.zpool.ca: Pool responded to subscribe
firopow.eu.mine.zpool.ca: Pool responded successfully to extranonce subscribe
firopow.eu.mine.zpool.ca: Pool responded successfully to authorize
Job 21 (kawpow)
Job 21 (kawpow)
GPU0 generating kawpow DAG (Epoch: 609)
```

**Comment by sp-hash** [comment]
When you get accepted shares you see firepow, but the job has the wrong algo displayed in the consolne. Will fix it in the next release.

**Comment by sp-hash** [comment]
fixed in v2.16 v2
